src/components/text-editor/examples/text-editor-with-inline-images-base64.tsx

```
@@ -0,0 +1,52 @@
+ * as base64 encoded data using the `insertImage` method.
+ *
+ * However, it is recommended to upload the image to an external file storage
+ * and insert the src url of the uploaded image, see the example above.
+ */
+@Component({
+    tag: 'limel-example-text-editor-with-inline-images-base64',
+    shadow: true,
+})
+export class TextEditorWithInlineImagesExample {
+    @State()
+    private value: string = 'Copy an image file and paste it here.';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)